Service/Manager.php

```diff
@@ -589,15 +589,20 @@ private function parseResult($raw, $resultsType, $scrollDuration = null)
      */
     private function convertToNormalizedArray($data)
     {
-        if (array_key_exists('_source', $data)) {
-            return $data['_source'];
+        if (array_key_exists('_source', $data) && array_key_exists('_id', $data)) {
```

Comment:
`_id` field always comes back in a response. No need to check for that. `_source` is checked in case of fields might be set in the query then `_source` won't come back.